src/platforms/python/guides/asgi/index.mdx

```diff
@@ -7,7 +7,11 @@ description: "Learn about using Sentry with ASGI applications."
 
 The ASGI middleware can be used to instrument any [ASGI](https://asgi.readthedocs.io/en/latest/)-compatible web framework to attach request data for your events.
 
-This can be used to instrument, for example [Starlette](https://www.starlette.io/middleware/) or [Django Channels 2.0](https://channels.readthedocs.io/en/latest/).
+<Note>
+
+If you use an ASGI framework please use our specific integrations for [FastAPI](/platforms/python/guides/fastapi/), [Starlette](/platforms/python/guides/starlette/), [Sanic](/platforms/python/guides/sanic/), [Quart](/platforms/python/guides/quart/), [Falcon](/platforms/python/guides/falcon/) instead of this ASGI middleware.
```

Comment:
```suggestion
If you use an ASGI framework, please use our specific integrations for [FastAPI](/platforms/python/guides/fastapi/), [Starlette](/platforms/python/guides/starlette/), [Sanic](/platforms/python/guides/sanic/), [Quart](/platforms/python/guides/quart/), and [Falcon](/platforms/python/guides/falcon/) instead of this ASGI middleware.
```